packages/grafana-e2e/cypress/plugins/index.js

```diff
@@ -12,6 +15,17 @@ module.exports = (on, config) => {
       return null;
     },
   });
+  on('task', {
+    getJSONFiles: async ({ dirPath }) => {
+      const directoryPath = path.join(__dirname, dirPath);
+      console.log(`Attempting to read contents of directory: ${directoryPath}`);
+      const jsonFiles = fs.readdirSync(directoryPath);
+      return jsonFiles.map((fileName) => {
```

Comment:
you probably want to filter out non json files in case the folder is shared with something else.

Reply:
there's not a great way of determining if a file is a json file or not but I'll add some error handling.

Reply:
well, I was thinking on just filtering out anything that doesn't have the extension `.json` (if anyone writes a JSON file and doesn't use the `.json` extension, that's asking for trouble :P). I wouldn't do the try-catch because if a file is meant to be used but it's malformed, it will return a false positive.

Reply:
ok cool! Updated.